INSTALL.md

```diff
@@ -198,7 +198,7 @@ yum install -y autoconf automake libtool pkgconfig fuse3-devel zlib-devel
 To download the source code do this:
 
 ```sh
-SQUASHFUSEVERSION=0.1.105
+SQUASHFUSEVERSION=0.1.1.6
```

Comment:
Please revert this line to 0.1.105

Reply:
Thanks for catching this. SQUASHFUSEVERSION is now set back to 0.1.105.